draft-thomson-webpush-http2.xml

```diff
@@ -78,29 +78,27 @@
   <middle>
     <section anchor="intro" title="Introduction">
       <t>
-        Mobile computing devices are increasingly relied upon for a great many
-        applications.  Mobile devices typically have limited power reserves, so
-        finding more efficient ways to serve application requirements is an
-        important part of any mobile platform.
+        Many applications on mobile and embedded devices require continuous
+        access to network communications so that real-time events - such as
+        incoming calls or messages - can be conveyed (or "pushed") to the user
+        in a timely fashion.
       </t>
-
-     <t>
-        One significant contributor to power usage mobile devices is the radio.
-        Radio communications consumes a significant portion of the energy budget
-        on a wirelessly connected mobile device.
+      <t>
+        Mobile and embedded devices typically have limited power reserves, so
+        finding more efficient ways to serve application requirements greatly
+        benefits the application ecosystem.  One significant contributor to
+        power usage is the radio. Radio communications consume a significant
+        portion of the energy budget on a wirelessly connected device.
       </t>
       <t>
-        Many applications require continuous access to network communications so
-        that real-time events - such as incoming calls or messages - can be
-        conveyed (or "pushed") to the user in a timely fashion.  Uncoordinated
-        use of persistent connections or sessions from multiple applications can
-        contribute to unnecessary use of the device radio, since each
-        independent session independently incurs overheads.  In particular, keep
-        alive traffic used to ensure that middleboxes do not prematurely time
-        out sessions, can result in significant waste.  Maintenance traffic
-        tends to dominate over the long term, since events are relatively rare.
+        Uncoordinated use of persistent connections or sessions from multiple
+        applications can contribute to unnecessary use of the device radio, 
+        since each independent session independently incurs overheads.  In
+        particular, keep alive traffic used to ensure that middleboxes do not
+        prematurely time out sessions, can result in significant waste.
+        Maintenance traffic tends to dominate over the long term, since events
+        are relatively rare.
```

Comment:
(For the last three paragraphs)

I think that the structural changes are OK here, but I'm not sure about changing the focus to embedded devices.  Maybe it would be good to instead talk about devices with limited battery capacity in general and avoid being specific.

I'm less concerned about the introduction parts than I am about some of the later changes.